include/lsst/ctrl/events/CommandEvent.h

```diff
@@ -63,38 +61,38 @@ namespace events {
 class CommandEvent : public Event
 {
 public:
-    static const std::string ORIGINATORID;
-    static const std::string ORIG_LOCALID;
+    static const std::string ORIG_HOSTNAME;
     static const std::string ORIG_PROCESSID;
-    static const std::string ORIG_IPID;
+    static const std::string ORIG_LOCALID;
 
-    static const std::string DESTINATIONID;
-    static const std::string DEST_LOCALID;
+    static const std::string DEST_HOSTNAME;
     static const std::string DEST_PROCESSID;
-    static const std::string DEST_IPID;
+    static const std::string DEST_LOCALID;
 
     CommandEvent();
-    CommandEvent(const std::string& runid, const int64_t originatorId, const int64_t destinationId, const PropertySet& ps);
-    CommandEvent(const std::string& runid, const int64_t originatorId, const int64_t destinationId, const PropertySet::Ptr psp);
+
+    CommandEvent(const LocationID& originator, const LocationID& destination, const PropertySet::Ptr psp);
```

Comment:
Please use `PTR(PropertySet) instead of `PropertySet::Ptr` (it makes it easier to forward-declare).
Our coding standard says the `const` should follow the type rather than precede it, e.g., `LocationID const& originator`.